.github/workflows/test.yml

```
@@ -21,60 +21,28 @@
         if: failure()
         uses: actions/upload-artifact@v4
         with:
           name: test-report
-          path: build/reports/tests/test
-
-      - name: Upload test coverage to Codecov
-        uses: codecov/codecov-action@v3
-        with:
-          files: build/reports/jacoco/test/jacocoTestReport.xml
-
+          path: build/reports/tests/test
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)